### ISSUE TITLE
When pasting a url with "__action" the query parameters behaves incorrectly

### ISSUE THREAD

**Issue by noam-honig** [rangav/thunder-client-support#914]
**Describe the bug**

**To Reproduce**

Paste the following url:
```
http://localhost:4200/test1/api/ActiveFamilyDeliveries?__action=liveQuery
```
See the Query Parameters, it doesn't display the "__action" query parameter. Instead, it shows a nonexistent "action" parameter.

![image](https://user-images.githubusercontent.com/16635859/207052976-62d23bbf-cccf-4e6d-bf66-20e51d5da044.png)

The same problem also happens with import curl

**Expected behavior**

**Platform:** 
 - OS: 
 - vscode version:
 - extension version:

**Your Team Size:**

**Comment by rangav** [comment]
@noam-honig Thanks for reporting the bug, will look into it.

**Comment by rangav** [comment]
This Bug is fixed now and published to marketplace, please update to `v2.2.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.2.0

Please let me know your feedback.

**Comment by noam-honig** [comment]
Thanks - you're the best
